metadata.rb

```diff
@@ -16,6 +16,8 @@
 depends 'compat_resource', '>= 12.14.6'
 depends 'zypper'
 
+suggests 'nssm'
```

Comment:
suggests was never fully implemented and we're working to fully deprecate the functionality in chef. This shouldn't be use here